### DISCUSSION TITLE
New keybinds for F-86 module

### DISCUSSION THREAD

**Discussion by correca** [Munkwolf/dcs-community-keybinds#34] [General]
Hi, these are new keybinds I use with F-86 module. Some pairs of commands I propose can be assigned to three-position switches on your game controller.

		-- Gear
		
		{down = iCommandPlaneGearUp, up = iCommandPlaneGearDown, name = _('Landing Gear Up else Down (2-way Switch)'), category = {_('Gear Systems'), _('Instrument Panel'), _('Custom')}},
		{down = iCommandPlaneGearDown, up = iCommandPlaneGearUp, name = _('Landing Gear Down else Up (2-way Switch)'), category = {_('Gear Systems'), _('Instrument Panel'), _('Custom')}},

		-- Air Brake

		{down = hydro_commands.F86_CockpitDeviceCommand_AirBrakesHandle, up = hydro_commands.F86_CockpitDeviceCommand_AirBrakesHandle, cockpit_device_id = devices.HYDRO_INTERFACE, value_down = -1, value_up = 0, name = _('Speed Brake Switch - OUT else HOLD (2-way Switch)'), category = {_('Flight Control'), _('Throttle Grip'), _('Custom')}},
		{down = hydro_commands.F86_CockpitDeviceCommand_AirBrakesHandle, up = hydro_commands.F86_CockpitDeviceCommand_AirBrakesHandle, cockpit_device_id = devices.HYDRO_INTERFACE, value_down = 1, value_up = 0, name = _('Speed Brake Switch - IN else HOLD (2-way Switch)'), category = {_('Flight Control'), _('Throttle Grip'), _('Custom')}},

		-- Left forward console

		{down = nav_lights_commands.F86_CockpitDeviceCommand_LandAndTaxiLights, up = nav_lights_commands.F86_CockpitDeviceCommand_LandAndTaxiLights, cockpit_device_id = devices.NAV_LIGHTS_INTERFACE, value_down = 1, value_up = 0, name = _('Landing & Taxi Lights Switch - EXTEND & ON else OFF (2-way Switch)'), category = {_('Left Forward Console'), _('Exterior Lighting'), _('Custom')}},
		{down = nav_lights_commands.F86_CockpitDeviceCommand_LandAndTaxiLights, up = nav_lights_commands.F86_CockpitDeviceCommand_LandAndTaxiLights, cockpit_device_id = devices.NAV_LIGHTS_INTERFACE, value_down = -1, value_up = 0, name = _('Landing & Taxi Lights Switch - RETRACT else OFF (2-way Switch)'), category = {_('Left Forward Console'), _('Exterior Lighting'), _('Custom')}},

		-- Right forward console

		{down = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLts, up = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLts,	cockpit_device_id = devices.NAV_LIGHTS_INTERFACE, value_down = 1, value_up = 0, name = _('Position and Fuselage Light Selector Switch - STEADY else OFF (2-way Switch)'), category = {_('Right Forward Console'), _('Exterior Lighting'), _('Custom')}},
		{down = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLts,	up = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLts,	cockpit_device_id = devices.NAV_LIGHTS_INTERFACE, value_down = -1, value_up = 0, name = _('Position and Fuselage Light Selector Switch - FLASH else OFF (2-way Switch)'), category = {_('Right Forward Console'), _('Exterior Lighting'), _('Custom')}},

		{down = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLtsDimmer, up = nav_lights_commands.F86_CockpitDeviceCommand_PosAndFusLtsDimmer, cockpit_device_id = devices.NAV_LIGHTS_INTERFACE, value_down = 1, value_up = -1, name = _('Exterior Lighting Dimmer Switch - BRIGHT else DIM (2-way Switch)'), category = {_('Right Forward Console'), _('Exterior Lighting'), _('Custom')}},

**Comment by Munkwolf** [comment]
Thank you! I just pushed an update for the Sabre with those keybinds added. I also went through the clickabledata.lua file for it again to bring the keybinds for it up to the standard of the other modules. 'Else' commands added for most everything, and a few circuit breakers didn't have binds available. I also added the Flaps handle as an axis, which works well with the slider on the Warthog (needs to be inverted in setup). I wish the other rotaries/dials like gun selector worked as axis binds, but they don't quantize the axis position to valid values like the flaps handle does.

**Reply by correca** [reply]
Amazing job mate! For now, as I map my new Warthog to different modules I'm adding some simple commands to use 2-way an 3-way switches. I'm sorry to send too many posts with new keybinds.
